github/comment_builder.go

```diff
@@ -0,0 +1,42 @@
+package github
+
+import (
+	"regexp"
+)
+
+type CommentBuilder struct {
```

Comment:
It's a bummer that you had to construct a whole new type to avoid the title vs. body splitting that is inherent in MessageBuilder. I'd like to help you avoid having to do this, but first I would have to refactor MessageBuilder a bit. Leave your approach as-is for now, please, until I think about how this can be done.